layabase/_api_mongo.py

```diff
@@ -24,7 +24,7 @@ def _add_query_field(
         # Describe every dict column field as dot notation
         for inner_field in field._default_description_model().__dict__.values():
             if isinstance(inner_field, Column):
-                _add_query_field(parser, inner_field, f"{field.name}.")
+                _add_query_field(parser, inner_field, f"{prefix}{field.name}.")
```

Comment:
test case is missing

Reply:
I already added a test.
test_open_api_definition
Without the fix it would show `first_key.inner_key1` with the fix it's showing the correct `dict_field.first_key.inner_key2`

Reply:
But the parser is also impacted if I am not mistaken ? Not only the open api définition

Reply:
Indeed I will add the tests right away

Reply:
Done adding a test which verifies that the multilevel dict fields are in the delete and get parser.
And adjusted the release date for today.